Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2069,8 +2066,12 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], params: D
             # Mirroring files as entries
             if entry.get('type') == 3:
                 path_res = demisto.getFilePath(entry.get('id'))
-                file_name = path_res.get('name')
-                client.upload_file(ticket_id, entry.get('id'), file_name, ticket_type)
+                full_file_name = path_res.get('name')
+                file_name, file_extension = os.path.splitext(full_file_name)
+                if not file_extension:
+                    file_extension = ''
+                client.upload_file(ticket_id, entry.get('id'), file_name + '_mirrored_from_xsoar' + file_extension,
```

Comment:
please add test for this to verify it is sent as expected (with the proper file name)